frontend/webservice/vrf.cgi

```diff
@@ -253,7 +253,22 @@ sub provision_vrf{
             return;
         }
 
+        my $last_octet = 2;
```

Comment:
This variable needs to go before the endpoint foreach just above... so that each endpoint also gets incremented

Reply:
Fixed in 9069ca3